XMonad/Actions/WindowBringer.hs

```diff
@@ -45,83 +46,106 @@ import XMonad.Util.NamedWindows (getName)
 -- For detailed instructions on editing your key bindings, see
 -- "XMonad.Doc.Extending#Editing_key_bindings".
 
--- | Default menu command
-defaultCmd :: String
-defaultCmd = "dmenu"
+data WindowBringerConfig = WindowBringerConfig { menuCommand :: String
```

Comment:
It would be good to include some Haddock documentation here, both on the data type itself and on the individual fields.  Maybe something like this:
```
-- | Configuration record for WindowBringer functions.
data WindowBringerConfig = WindowBringerConfig
  { menuCommand   :: String     -- ^ The shell command used to xyz.
  , menuArgs      :: [String]   -- ^ Foo bar
  ...
```

I personally like to put the opening brace indented on a separate line like this, it results in more consistent indentation and leaves more horizontal space for including comments and such.